src/main/java/com/artipie/asto/fs/FileStorage.java

```diff
@@ -205,6 +207,7 @@ public FileStorage(final Path path, final ExecutorService exec) {
             () -> {
                 try {
                     Files.delete(this.path(key));
```

Comment:
@olenagerasimova I think we need to add a check that path of key we're going to remove here is not a directory for consistency with `exists` method